### ISSUE TITLE
uninitialized constant SidekiqClientCLI::CLI

### ISSUE THREAD

**Issue by rileys** [didil/sidekiq-client-cli#17]
Using version 0.1.7, sidekiq 3.4.2

`bundle exec sidekiq-client push ScheduleCurrentPostsWorker`
`
uninitialized constant SidekiqClientCLI::CLI
/home/vagrant/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/sidekiq-client-cli-0.1.7/lib/sidekiq_client_cli.rb:12:in `parse'
/home/vagrant/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/sidekiq-client-cli-0.1.7/bin/sidekiq-client:7:in `<top (required)>'
/home/vagrant/.rbenv/versions/2.2.2/bin/sidekiq-client:23:in `load'
/home/vagrant/.rbenv/versions/2.2.2/bin/sidekiq-client:23:in `<main>'
`

**Comment by didil** [comment]
tried to reproduce your environment but I can't reproduce the error you're seeing, seems like a missing dependency for the cli gem , could you try running bundle install again ? are you still seeing the error ?

**Comment by rileys** [comment]
Yes tried bundle again and no luck.  I decided to use the 'sidekiq_pusher' script recommended on the sidekiq wiki, and seems to work fine.

**Comment by didil** [comment]
ok great you found an alternative solution. I'll leave this issue open for a while if anybody else is having the same issue please comment here then we can dig deeper but right now I can't seem to reproduce it locally or on Travis CI
